libobs/util/platform-windows.c

```diff
@@ -975,6 +975,24 @@ bool is_64_bit_windows(void)
 #endif
 }
 
+bool is_arm64_windows(void)
+{
+	USHORT _, nativeMachine;
+	auto result = IsWow64Process2(GetCurrentProcess(), &_, &nativeMachine);
+	return (result && (nativeMachine == IMAGE_FILE_MACHINE_ARM64));
+}
+
+bool os_get_emulation_status()
+{
+#if defined(_M_ARM64) || defined(_M_ARM64EC)
+	return false;
+#else
+	if (is_arm64_windows()) {
```

Comment:
Is there a reason for this being a separate function? Doesn’t *really* matter, but could probably be inlined.

Reply:
I got your point, I was equally confused when I wrote this. 

If you look a few lines above, you'll see a function `is_64_bit_windows()` which have the similar purpose. 

These 2 functions serve 2 different purpose but can have exactly the same implementation.